src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonResourceIntTest.kt

```diff
@@ -29,6 +31,12 @@ class ContactPersonResourceIntTest : IntegrationTestBase() {
   @Autowired
   private lateinit var personRepository: PersonRepository
 
+  @Autowired
+  private lateinit var personContactRepository: OffenderContactPersonRepository
```

Comment:
minor - everything else seems to be contactPerson rather than personContact

Reply:
Yes the "domain" os Person Contact but this specific entity is the Contact Person.

Yes _ I know - hell - but Person Contact is umbrella for about 9 different entities